robot/src/main/java/us/ilite/robot/auto/paths/FieldElementLocations.java

```diff
@@ -12,7 +12,7 @@
     //------------------------------
 
     // Two middle hatches
-    public static final Translation2d kCargoShipMiddleLeftHatch = new Translation2d(150.0, 150.0);
+    public static final Translation2d kCargoShipMiddleLeftHatch = new Translation2d(18.5 * 12.0 , 12.0 * 12.0);
```

Comment:
What does this math mean? In other words: what does 18.5 x 12 and 12 x 12 mean?

Reply:
This is an (x, y) position